Elcano_C2_LowLevel/Elcano_C2_LowLevel.ino

```diff
@@ -236,119 +263,156 @@ void setup()
   //brake(MIN_BRAKE_OUT);
   Serial.begin(9600);
   rc_index = 0;
-  for (int i = 0; i < 8; i++)
+  for (int i = 0; i < RC_NUM_SIGNALS; i++)
   {
     RC_rise[i] = INVALID_DATA;
     RC_elapsed[i] = INVALID_DATA;
+    RC_Done[i] = 0;
   }
   for (int i = 0; i < ERROR_HISTORY; i++)
   {
     speed_errors[i] = 0;
   }
-  //setupWheelRev(); // WheelRev4 addition
-  CalibrateTurnAngle(32, 20);
```

Comment:
I'm hoping that the folks who worked on it will comment.  Is this intended to run on every startup?  Is it intended to overcome bad hardware, or is this something we expect to need even with properly functioning hardware?  Didn't we conclude that the sensor is bad, or the mount is loose, and should be replaced?  If the sensor is turning around systematically, that's "drift".  If the clamp onto the shaft is sloppy, so that it has hysteresis, that's different.  Both are bad, but drift is especially bad, and worse if it's happening as the trike is in operation.  At least hysteresis doesn't continually shift in one direction, or shift arbitrarily or randomly.